src/referentielijsten/api/admin.py

```diff
@@ -11,6 +11,11 @@
 from .models import Item, Tabel
 
 
+class ItemInline(admin.StackedInline):
+    model = Item
+    extra = 1
+
```

Comment:
```suggestion
    fields = (
        "code",
        "naam",
        "begindatum_geldigheid",
        "einddatum_geldigheid",
        "aanvullende_gegevens",
    )
```
Lets add fields to keep it consistent with the other admin classes